### PR TITLE
[Flake Test] scheduler(queue): fix flake test for InFlightPods

### PR DESCRIPTION
**Pull request by googs1025**
#### What type of PR is this?
/kind flake
/kind cleanup

#### What this PR does / why we need it:
- Maybe we need to determine the order before comparing
#### Which issue(s) this PR fixes:

Fixes # https://github.com/kubernetes/kubernetes/issues/126801

#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
None
```

### CODE REVIEW

**Comment by googs1025** [comment]
@macsko @sanposhiho @pacoxu /PTAL thanks

**Comment by googs1025** [comment]
/assign @macsko @sanposhiho

Can we deal with this issue as soon as possible? :)

File: pkg/scheduler/internal/queue/scheduling_queue_test.go [hunk 1]
```diff
@@ -734,7 +734,8 @@ func Test_InFlightPods(t *testing.T) {
 				for _, pod := range pods {
 					podNames = append(podNames, pod.Name)
 				}
-				if diff := cmp.Diff(test.wantActiveQPodNames, podNames); diff != "" {
+				sortOpt := cmpopts.SortSlices(func(a, b string) bool { return a < b })
```
Comment (sanposhiho) [inline pkg/scheduler/internal/queue/scheduling_queue_test.go:737]: can we move sortOpt somewhere else so that it's declared only once.

Comment (googs1025) [reply pkg/scheduler/internal/queue/scheduling_queue_test.go]: done

**Review by sanposhiho — APPROVED** [review]
/lgtm
/approve

**Pull request merged**